redisinsight/desktop/src/config/configMain.ts

```diff
@@ -7,6 +7,9 @@ import pkg from '../../../package.json'
 const config: any = configInit
 
 // Merge in some details from package.json
+config.build = process.env.NODE_ENV === 'development' ? 'development' : 'production'
+config.devBuild = process.env.NODE_ENV === 'development'
```

Comment:
you added this field and don't use it
Also there is the same `config.isDevelopment`. Please use only 1 field `isDevelopment` or rename it to `devBuild`